frontend/src/routes/candidate/(protected)/profile/+page.svelte

```diff
@@ -15,8 +19,60 @@
     ['motherTongue', ['English', 'Suomi', 'Svenska']]
   ]);
 
-  const labelClass = 'label-sm label mx-6 my-2 text-secondary';
-  const disclaimerClass = 'm-0 p-0 text-sm text-secondary';
+  const labelClass = 'w-6/12 label-sm label mx-6 my-2 text-secondary';
+  const disclaimerClass = 'mx-6 my-0 p-0 text-sm text-secondary';
+  const headerClass = 'mx-6 my-0 p-0 text-m text-secondary';
+  const inputClass =
+    'input-ghost input input-sm w-full pr-2 text-right disabled:border-none disabled:bg-base-100';
+
+  let portraitInput: HTMLElement | null;
+  let portraitLabel: HTMLElement | null;
+
+  // function for clicking the portrait input field with space
+  const handlePortraitInput = (event: KeyboardEvent) => {
+    if (event.code === 'Space') {
+      event.preventDefault(); // Prevent default behavior (e.g., scrolling the page)
+      portraitInput?.click();
+    }
+  };
+
+  // add an event listener to the portrait label for keyboard navigation
+  onMount(() => {
+    portraitInput = document.getElementById('portrait');
+    portraitLabel = document.getElementById('portraitLabel');
+
+    portraitLabel?.addEventListener('keydown', handlePortraitInput);
+  });
+
+  // remove the event listener on unmount
+  onDestroy(() => {
+    portraitLabel?.removeEventListener('keydown', handlePortraitInput);
+  });
+
+  // get the user from authContext
+  const user = get(authContext.user);
+  const nominations = user?.candidate.nominations;
+
+  // the dot symbol for separating info string
+  const dot = '\u22C5';
+
+  // map nominations into objects
+  const nominationFields = nominations?.map((nom) => ({
+    nominationID: nom.id,
+    constituency: nom.constituency.name,
+    party: nom.party.shortName,
+    electionSymbol: nom.electionSymbol,
+    fieldText: nom.electionSymbol
+      ? `${nom.constituency.name} ${dot} ${nom.party.shortName} ${dot} ${nom.electionSymbol}`
+      : `${nom.constituency.name} ${dot} ${nom.party.shortName}`
```

Comment:
This can be shortened:
```
fieldText: `${nom.constituency.name} ${dot} ${nom.party.shortName} ${nom.electionSymbol ? dot + ' ' + nom.electionSymbol : ''}`
```